src/section_handlers/document_tracking.py

```diff
@@ -0,0 +1,154 @@
+                }
+            )
+
+        else:
+            raise ValueError('Can not handle such version difference between The current and the history"s last version')
+
+
+    def _reindex_history_to_integers(self, root_element, revision_history):
+        logging.info('Some version numbers in /document/tracking/revision_history do not match semantic versioning. '
+                     'Reindexing to integers.')
+
+        revision_history_sorted = sorted(revision_history, key=itemgetter('version_as_int_tuple'))
+
+        for rev_number, revision in enumerate(revision_history_sorted, start=1):
+            revision['number'] = rev_number  # Changing the type from str to int
+
+        # after reindexing, match document version to corresponding one in revision history
+        version = next(rev for rev in revision_history_sorted if rev['number_cvrf'] == root_element.Version.text)['number']
+
+        return revision_history_sorted, version
+
+    def _process_revision_history(self, root_element):
+        # preprocess the data
+        revision_history = []
+        for revision in root_element.RevisionHistory.Revision:
+            # number_cvrf: keep original value in this variable for matching later
+            # number: this value might be overwritten later if some version numbers doesn't match semantic versioning
+            revision_history.append(
+                {
+                    'date': revision.Date.text,
+                    'number': revision.Number.text,
+                    'summary': revision.Description.text,
+                    # Extra vars
+                    'number_cvrf': revision.Number.text,
+                    'version_as_int_tuple': self._as_int_tuple(revision.Number.text),
+                }
+            )
+
+        # Do we miss the current version in the revision history?
+        if not [rev for rev in revision_history if rev['number'] == root_element.Version.text]:
+            try:
+                self._add_current_revision_to_history(root_element, revision_history)
+                logging.warning(f'/document/tracking/version value was not found in /document/tracking/revision_history. Adding the current revision to the history')
+            except ValueError as e:
+                # TODO: What to do here? If the current version is not semver OR the difference between versions is > 0.1.0
+                logging.critical(f'/document/tracking/version value was not found in /document/tracking/revision_history. Converter doesn"t know how to proceed. Reason: {e}')
+                exit(1)
+
+        # handle corresponding part of Conformance Clause 5: CVRF CSAF converter
+        # that is: some version numbers in revision_history don't match semantic versioning
+        if not self.check_for_version_t(revision_history):
+            # TODO: X.Y is not considered a valid semantic versioning, shall we assume X.Y.0 here?
```

Comment:
Definitely not. Please see my [comment](https://github.com/csaf-tools/CVRF-CSAF-Converter/pull/21#issuecomment-1004249280): X.Y is always converted to integer versioning.